spec/lib/reporting_spec.rb

```diff
@@ -40,8 +40,8 @@
         let(:whitespace_note) { user1.notes.create(body: " \n \n \n") }
 
         it 'the note is not counted' do
-          expect(blank_note.filled?).to eq(false)
-          expect(whitespace_note.filled?).to eq(false)
+          blank_note
```

Comment:
`let` is lazy and loaded on demand. The alternative is `let!` which creates them there and then. If that isn't suitable I'd always use the `before` block for prerequisite steps, because perhaps some can be shared.

Reply:
I could use `let!` for these but since they aren't explicitly referenced in the tests, rubocop is unhappy. Currently just referencing the notes in the before block to load them

Reply:
As we've seen in the other stats PR we can just create notes in the before block and reduce the number of lines.

Reply:
[80d99bf](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/645/commits/80d99bfa3c5798dd8cca37000444dfc237b1cf0f)